flink-kubernetes/src/main/java/org/apache/flink/kubernetes/kubeclient/KubernetesJobManagerSpecification.java

```diff
@@ -16,19 +16,32 @@
  * limitations under the License.
  */
 
-package org.apache.flink.kubernetes.kubeclient.resources;
-
-import org.apache.flink.configuration.Configuration;
+package org.apache.flink.kubernetes.kubeclient;
 
+import io.fabric8.kubernetes.api.model.HasMetadata;
 import io.fabric8.kubernetes.api.model.apps.Deployment;
-import io.fabric8.kubernetes.api.model.apps.DeploymentBuilder;
+
+import java.util.List;
 
 /**
- * Represent KubernetesDeployment resource in kubernetes.
+ *	Composition of the created Kubernetes components that represents a Flink application.
```

Comment:
```suggestion
 *  Composition of the created Kubernetes components that represents a Flink application.
```

Reply:
You can edit the commit introduce this instead of adding fixup. I think we agree on most of codes now.